storybook/story-components/story-page.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```

Comment:
It'd be good to have a native version of this that at minimum adds a header or a top margin, or centers the component in the screen. On real android, the component clashes with the device header: 

<img src="https://user-images.githubusercontent.com/29628323/87419884-2dc35e00-c5cc-11ea-8b1e-9575694c99b6.jpg" width=440 />